docusaurus/docs/reference/ha/certificates.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_label: Certificates
+sidebar_position: 20
+---
+
+# Controller Certificates
+
+For controllers to communicate and trust one another, they need certificates that have
+been generated with the correct attribute and relationships.
+
+## Requirements
+
+1. The certificates must have a shared root of trust
+2. The controller client and server certificates must contain a 
+   [SPIFFE ID](https://spiffe.io/docs/latest/spiffe-about/spiffe-concepts/#spiffe-id)
+
+## Steps to Certificate Creation
+There are many ways to set up certificates, so this will just cover a recommended configuration.
```

Comment:
> recommended configuration 

is such a loaded term. I'd rather remove that and refer to this as an example configuration. Or -- with all the spec-reading I've been doing -- use the words "non-normative"... and feel fancy about it.... :)

Reply:
switched to example